packages/esm-patient-chart-app/src/root.component.tsx

```diff
@@ -44,6 +44,13 @@ export default function Root() {
  * t('error', 'Error')
  * t('seeAll', 'See all')
  * t('paginationItemsCount', `{{pageItemsCount}} / {{count}} items`, { count: totalItems, pageItemsCount });
- * t('unsavedChangesInForms', 'There are unsaved changes in the following workspaces. Do you want to discard changes in the following workspaces? {{workspaceNames}}')
- * t('closeWorkspaces', 'Discard changes in {{count}} workspaces', {count})
+
+ * t('unsavedChangesInOpenedWorkspace', 'You have unsaved changes in the opened workspace. Do you want to discard these changes?')
+ * t('unsavedChangesTitleText', 'Unsaved Changes')
+ * t('discard','Discard')
+ * t('closingAllWorkspacesPromptTitle','You have unsaved changes')
+ * t('closingAllWorkspacesPromptBody', 'There are unsaved changes in the following workspaces. Do you want to discard changes in the following workspaces? {{workspaceNames}}')
+ * t('closeAllOpenedWorkspaces', 'Discard changes in {{count}} workspaces', {count})
+ * t('unsavedChangesInWorkspace', 'There are unsaved changes in {{workspaceName}}. Please save them before opening another workspace.')
```

Comment:
Nit: I'm not sure that relying on workspace names for this particular translation is the most ideal approach because they're so arbitrary. That said, I don't have a better alternative.

Reply:
Since we had this in the previous versions as well, I say we move ahead with this